src/service/main.ts

```diff
@@ -106,34 +106,33 @@ var messagingLog = Utils.log("tribeca:messaging");
 
 function ParseCurrencyPair(raw: string) : Models.CurrencyPair {
     var split = raw.split("/");
-    if (split.length !== 2) 
+    if (split.length !== 2)
         throw new Error("Invalid currency pair! Must be in the format of BASE/QUOTE, eg BTC/USD");
-    
+
     return new Models.CurrencyPair(Models.Currency[split[0]], Models.Currency[split[1]]);
 }
 var pair = ParseCurrencyPair(config.GetString("TradedPair"));
 
 var defaultActive : Models.SerializedQuotesActive = new Models.SerializedQuotesActive(false, moment.unix(1));
-var defaultQuotingParameters : Models.QuotingParameters = new Models.QuotingParameters(.3, .05, Models.QuotingMode.Top, 
-    Models.FairValueModel.BBO, 3, .8, false, Models.AutoPositionMode.Off, false, 2.5, 300, .095, 2*.095, .095, 3, .1);
+var defaultQuotingParameters : Models.QuotingParameters = new Models.QuotingParameters(2, 0.02, 0.01, Models.QuotingMode.Boomerang, Models.FairValueModel.BBO, 1, 0.9, true, Models.AutoPositionMode.EwmaBasic, false, 0.9, 569, false, .095, 2*.095, .095, 3, .1);
```

Comment:
im sorry i had deffined new defaults, you can set your own back as you wish ofcourse